issuetracker/internal/youtrack/task.go

```diff
@@ -16,20 +17,24 @@ type Task struct {
 }
 
 // GetStatus of youtrack task, based on underlying structure
-func (t *Task) GetStatus() taskstatus.TaskStatus {
+func (t *Task) GetStatus() (taskstatus.TaskStatus, error) {
 	isResolved := false
+	var statusError error
 	// Loop through all fields to find status field
 	for _, node := range t.CustomFields {
-		if node[Type].(string) == StateType && node[Value] != nil {
+		if node[Type].(string) == StateType {
+			if node[Value] == nil {
+				statusError = errors.New("couldn't fetch issue status from youtrack. This is probably on us, please file a bug report here: https://github.com/preslavmihaylov/todocheck/issues/new?title=failed%20get%20youtrack%20issue%20status&labels=bug")
+			}
```

Comment:
just return directly from here. It's much easier than assigning this to a variable & passing it later.

On the other places, `return taskstatus.Closed/Open, nil`